databuilder/models/table_metadata.py

```diff
@@ -159,11 +189,7 @@ def __init__(self,
         self.columns = columns if columns else []
         self.is_view = is_view
         self.attrs = None
-        if isinstance(tags, str):
-            tags = list(filter(None, tags.split(',')))
-        if isinstance(tags, list):
-            tags = [tag.lower().strip() for tag in tags]
-        self.tags = tags
+        self.tags = TagMetadata.parse_tags_from_strings(tags)
```

Comment:
Could change so the type is either strings or a list of TagMetadata. This way don't have to have the ^b convention except for inside the csv_table_column_extractor

Reply:
sounds good